src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -252,6 +256,29 @@ def nlcd_streams(sjs_result):
 
 
 @shared_task(throws=Exception)
+def nlcd_streams_drb(sjs_result):
+    """
+    This callback is run when the geometry falls within the DRB. We calculate
+    the percentage of DRB streams in each land use type.
+    """
+    if 'error' in sjs_result:
+        raise Exception('[nlcd_streams_drb] {}'.format(sjs_result['error']))
+
+    result = parse_sjs_result(sjs_result)
+    total = sum(result.values())
+
+    lu_stream_pct_drb = [0.0] * 16
+    for nlcd, stream_count in result.iteritems():
+        lu = get_lu_index(nlcd)
+        if lu is not None:
+            lu_stream_pct_drb[lu] += float(stream_count) / total
```

Comment:
Although we probably don't need to, because if `total` has a value it is because some item in result has a value. Thus, when `total` is 0, we simply won't enter the `for nlcd, stream_count in result.iteritems()` loop.